index.js

```diff
@@ -56,6 +56,25 @@ module.exports = {
     'import/no-named-default': 'error',
     'import/no-webpack-loader-syntax': 'error',
     'import/prefer-default-export': 'error',
+    'jsdoc/check-examples': 0,
+    'jsdoc/check-param-names': 1,
+    'jsdoc/check-tag-names': 1,
+    'jsdoc/check-types': 1,
+    'jsdoc/newline-after-description': 0,
+    'jsdoc/no-undefined-types': 0, // did not consider InputValidationError as valid type
```

Comment:
If it doesn't recognize required types as `const Type = require("./type")`, then indeed we should keep it off

Reply:
an example from backend/webhooks/webhooks-payments/index.js. The code seems valid:
```
...
const InputValidationError = require('../../lib/errors/InputValidationError');
...
 * @throws {InputValidationError} If signature is invalid
```
result: [eslint] The type 'InputValidationError' is undefined. [jsdoc/no-undefined-types]

Reply:
Wierd, rules seems to not work with NCJS requires, therefore indeed we can't rely on it.